app/components/sidebars/main/channels_list/channel_item/channel_item.js

```diff
@@ -22,7 +22,7 @@ export default class ChannelItem extends PureComponent {
         channelId: PropTypes.string.isRequired,
         channel: PropTypes.object,
         currentChannelId: PropTypes.string.isRequired,
-        displayName: PropTypes.string.isRequired,
+        displayName: PropTypes.string,
```

Comment:
Why not required anymore?

Reply:
@enahum It was causing an error when cache is deleted. Also on line 93, there's a check for 
```
        if (!displayName) {
            return null;
        }
```

which to me didn't look like it's required if it's checking that.

Reply:
nice catch!